sections/featured-product.liquid

```diff
@@ -284,9 +283,19 @@
                 {% assign share_url = product.selected_variant.url | default: product.url | prepend: request.origin %}
                 {% render 'share-button', block: block, share_link: share_url %}
               {%- when 'variant_picker' -%}
-                {% render 'product-variant-picker', product: product, block: block, product_form_id: product_form_id, update_url: false %}
+                {% render 'product-variant-picker',
+                  product: product,
+                  block: block,
+                  product_form_id: product_form_id,
+                  update_url: false
+                %}
               {%- when 'buy_buttons' -%}
-                {%- render 'buy-buttons', block: block, product: product, product_form_id: product_form_id, section_id: section.id -%}
+                {%- render 'buy-buttons',
+                  block: block,
+                  product: product,
+                  product_form_id: product_form_id,
+                  section_id: section.id
+                -%}
```

Comment:
There are a bunch of little reformatting additions like these that make it harder to review. None of it appears to actually change anything though.

Reply:
Yeah 😞 I will add a note in the PR description to mention this. We're trying to default to use our own prettier extension  and I guess some stuff wasn't formatted in some files.